src/app/movies/movies.component.ts

```diff
@@ -21,6 +22,8 @@ export class MoviesComponent {
   }
 
   parseUserInput(movieInput: string): Movie {
+    const getVideoMetadata = getVideoId.getVideoId(movieInput);
```

Comment:
zobacz no głupio to wygląda getVideoId.getVideoId - tak jak Ci wcześniej pisałem, zrób `import { getVideoId } from '../shared/recognize-video-type-id'; i wtedy będziesz mógł używać jako getVideoId.getVideoId.
Kiedy piszesz import * as cośtam from gdzieśtam, to importujesz wszystkie eksporty z pliku gdzieśtam, ładujesz je do obiektu który nazwany jest cośtam. I potem musisz tego używać jako cośtam.imięFunkcji. Dlatego zazwyczaj importuje się jako import { imięFunkcji } from gdzieśtam; i używa jako od razu imięFunkcji ;)

Reply:
Ok poprawione działa.